src/uu/df/src/table.rs

```diff
@@ -212,25 +212,57 @@ impl<'a> DisplayRow<'a> {
         Self { row, options }
     }
 
-    /// Get a string giving the scaled version of the input number.
+    /// Get a human readable string giving the scaled version of the input number.
     ///
     /// The scaling factor is defined in the `options` field.
     ///
+    /// This function is meant to be used by scaled_bytes() resp. scaled_inodes() only.
```

Comment:
I don't understand the use of "resp." in this sentence. Could you rephrase it?

Reply:
Ok, I rephrased it. I hope it's an improvement.